products/redhat-jboss-eap.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Red Hat JBoss Enterprise Application Platform
+category: server-app
+iconSlug: redhat
+permalink: /redhat-jboss-eap
+alternate_urls:
+-   /redhat-jboss-eap
+-   /redhatjboss
+-   /redhatjbosseap
+
+versionCommand: $JBOSS_HOME/bin/version.sh
+releasePolicyLink: https://access.redhat.com/support/policy/updates/jboss_notes#phases
+activeSupportColumn: Full Support
+eolColumn: Maintenance Support
+releaseDateColumn: true
+extendedSupportColumn: Extended Life Cycle Support
+
+releases:
+-   releaseCycle: "7"
+    support: 2023-06-30
+    eol: 2025-06-30
+    extendedSupport: 2029-11-30
+    latest: "7.4.6"
+    releaseDate: 2016-05-01
+    lts: 2026-11-30
+    latestReleaseDate: 2023-03-01
+    link: https://access.redhat.com/articles/6716841
+
+-   releaseCycle: "6"
+    support: 2016-06-30
+    eol: 2019-06-30
+    extendedSupport: 2025-06-30
+    latest: "6.4"
```

Comment:
6.4.10 according the the link below (https://access.redhat.com/articles/3099701) ?

Reply:
https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/6.4

[`6.4.24`](https://access.redhat.com/articles/6958003)